### ISSUE TITLE
Allow output checkers to submit review from popup

### ISSUE THREAD

**Issue by inglesp** [opensafely-core/airlock#704]
When an output checker has finished their review they are shown a popup telling them they need to submit their review.  We should allow them to do this from the popup itself.

**Comment by inglesp** [comment]
I don't think this quite as straightforward as we first thought.  If we want to show action links in the popup, why have them in the request summary as well?  The popup will be shown if and only if any of the actions are possible.

![Image](https://github.com/user-attachments/assets/e7069c73-3c3a-4fa6-ab7d-ed7bf075227f)

**Comment by rw251** [comment]
PR #828 possibly makes this redundant.

**Comment by rebkwok** [comment]
I think this is done (as much as it can be). There's a link from the alter to go to the overview page to submit the review. Actually submitting from the modal is tricky and I deliberately decided not to do it.
